### PR TITLE
Spelling and grammar changes

### PR DESCRIPTION
**Pull request by Jordan-Nguy**
Spelling and grammar fixes

### CODE REVIEW

File: docs/vote-escrow/realms.mdx [hunk 1]
```diff
@@ -157,7 +157,7 @@ then it declines linearly until release.
 
 ### Duration
 
-The duration of your cooldown is defined in days. The longer your tokens are staked, the more veHNT
+The duration of your cooldown is defined as days. The longer your tokens are staked, the more veHNT
```
Comment (jthiller) [inline docs/vote-escrow/realms.mdx]:
```suggestion
The duration of your cooldown is set in units of days. The longer your tokens are staked, the more veHNT
```
How about something like this?

Comment (Jordan-Nguy) [reply docs/vote-escrow/realms.mdx]: Too wordy imo, most of the people reading probably fly through anyways.

**Review by jthiller — APPROVED** [review]
